### DISCUSSION TITLE
Icon Request: warehouse-empty

### DISCUSSION THREAD

**Discussion by Pettrie** [FortAwesome/Font-Awesome#21214] [Icon Requests]
In group Logistics, add an empty warehouse

**Comment by rent-is** [comment]
This would be a good/valuable additional icon!

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
